frontend/src/components/ConfigurationP1.tsx

```diff
@@ -0,0 +1,207 @@
+    toggleHubHitoricalRefLinksImport: ()=>void;
+    toggleExpandAccordion: ()=>void;
+  }
+}
+
+export function ConfigurationP1() {
+  return <View {...Controller()}/>
+}
+
+export function Controller(){
+  const emptyConfig: HubConfig = {
+    createContact: false,
+    importHistoricalContacts: false,
+    deleteContact: false,
+    syncRefLinks: false,
+    importHistoricalRefLinks: false,
+  }
+  const [config, setConfig] = useState<HubConfig>(emptyConfig)
+  const [expandAccordion, setExpandAccordion] = useState<boolean>(false)
+
+  // Need a handler for each toggle because Switches are kinda weird
+  const toggleHubCreate = () => {
+    if (!expandAccordion && !config.createContact) {
+      setExpandAccordion(true)
+    }
+    setConfig({...config, createContact: !config.createContact});
+  }
+  const toggleHubHistoricalImport = () => setConfig({...config, importHistoricalContacts: !config.importHistoricalContacts});
+  const toggleHubDelete = () => setConfig({...config, deleteContact: !config.deleteContact});
+  const toggleHubRefLinks = () => setConfig({...config, syncRefLinks: !config.syncRefLinks});
+  const toggleHubHitoricalRefLinksImport = () => setConfig({...config, importHistoricalRefLinks: !config.importHistoricalRefLinks});
+  const toggleExpandAccordion = () => setExpandAccordion(!expandAccordion);
+
+  const handleToggles = {
+    toggleHubCreate,
+    toggleHubHistoricalImport,
+    toggleHubDelete,
+    toggleHubRefLinks,
+    toggleHubHitoricalRefLinksImport,
+    toggleExpandAccordion,
+  };
+  
+  const handleSubmit = () => {
+    // Here we will make the requests to the backend to store the config info and to begin the integration and redirect to second config screen
+    history.push('/configuration/2');
+  }
+  return {config, expandAccordion, handleSubmit, handleToggles} as states
+}
+
+export function View(states: states){
+  return (
+    <PageWrapper>
+      <PageContent>
+        <TitleText>Configure your <Logo src={HubspotLogo} /> Integration</TitleText>
+        <UnpaddedAccordion expanded={states.expandAccordion} onChange={states.handleToggles.toggleExpandAccordion}>
+          <AccordionSummary expandIcon={<ExpandMoreIcon />}>
+            <ToggleSetting 
+              settingText={"Create a Contact in HubSpot when a new user is added to SaaSquatch"} 
+              isChecked={states.config.createContact} 
+              handleChange={states.handleToggles.toggleHubCreate} 
+            />
+          </AccordionSummary>
+          <AccordionDetails>
+            <AccordionDetailsContainer>
+              <InfoText>A contact will be created in Hubspot when a new participant is created in SaaSquatch with the selected fields</InfoText>
+              <DataGrid 
+                rows={[
+                  { id: 1, fieldName: 'First name',},
+                  { id: 2, fieldName: 'Last name',},
+                  { id: 3, fieldName: 'Email',},
+                  { id: 4, fieldName: 'other stuff',},
```

Comment:
Can we add "Contact Owner", "Associated Company", "Last Activity Date", and "Create Date" here instead of other stuff?